sqlpad/README.md

```diff
@@ -22,11 +30,20 @@ Google sign-on has been configured for the application and only users invited by
 
 `var.sqlpad_admin` - Email address for the initial admin user, once logged in additional users can be configured.
 
-#### Google SSO
-`var.sqlpad_google_client_id` - Google Client Id for SSO
-`var.sqlpad_google_client_secret` - Google Client Secret for SSO
+`sqlpad_admin_password` - in az keyvault
+
+#### Google SSO (az keyvault)
+`sqlpad_google_client_id` - Google Client Id for SSO
+
+`sqlpad_google_client_secret` - Google Client Secret for SSO
+
+These are generated in the Google Cloud 'Becoming a Teacher' project
 
 #### Postgres connection details
+
+For prod, connections are only created by the admin after the initial app build.
```

Comment:
Can we discuss with the team so we rebuild QA the right way now? @markhallen

Reply:
as discussed we will leave qa as is for now, and removed qa workspace_variables